.github/workflows/preview.yml

```
@@ -47,9 +49,10 @@ jobs:
 
     - name: Ingest and Render
       run: |
+        papyri ingest  --no-relink ~/.papyri/data/papyri_0.0.8
         papyri ingest  --no-relink ~/.papyri/data/*
         papyri relink
-        ls -al
+        papyri render --ascii
```

It was not you, it was me, I probably tried to just remove `--ascii`.
```suggestion
        papyri render
```